qutebrowser/config/configfiles.py

```diff
@@ -553,6 +554,23 @@ def _migrate_bindings_default(self) -> None:
         del self._settings['bindings.default']
         self.changed.emit()
 
+    def _migrate_hints_uppercase(self) -> None:
```

Comment:
This looks good from what I can see, but still needs some tests. See e.g. `test_font_default_family` in `tests/unit/config/test_configfiles.py` for some inspiration.